tests/stdlib/tlists.nim

```diff
@@ -2,15 +2,15 @@ discard """
   targets: "c js"
 """
 
```

Comment:
```suggestion
  targets: "c"
"""
#XXX: js target currently disabled for this test because `swap a, b` call at lists.nim:408 has no effect
```

Reply:
@salvipeter this is bad for 2 reasons:
* it's a regression (previously js was being tested, now it's not tested at all and won't catch future regressions); when you encounter such cases, instead of disabling the entire test, disable just what's needed, as follows:
```nim
block: # prepend, prependMoved
  when not defined(js): # xxx swap broken in js, refs: issue #xyz
    ...
```

* furthermore, the issue is actually easy to work around, simply write:
```nim
# in lists.prependMoved:
  when defined(js): # xxx swap broken in js, refs: issue #xyz`
    var a2 = a
    a = b
    b = a2
  else:
    swap a, b
```

Reply:
@timotheecour It was me who commited the suggestion, not salvipeter.
If it's easy to workaround, sure let's do that.

Reply:
Done